srcpkgs/quiterss/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'quiterss'
+pkgname=quiterss
+version=0.19.4
+revision=1
+build_style=qmake
+hostmakedepends="pkg-config qt5-qmake"
+makedepends="sqlite-devel qt5-webkit-devel qt5-multimedia-devel"
+short_desc="Open-source cross-platform RSS/Atom news feeds reader"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://quiterss.org/"
+distfiles="https://github.com/QuiteRSS/${pkgname}/archive/${version}.tar.gz"
+checksum=d9dffa205a8ec4e7bf00d87183fd94d4a12f045fae04a6efd41d6557827233bc
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-webkit-devel qt5-multimedia-devel"
```

Comment:
In my _very opinionated and probably incorrect_ opinions, I'll put those things into `hostmakedepends` regardless of `$CROSS_BUILD`

Reply:
Wouldn't that make building non-cross slower, because of the additional dependencies?

Reply:
They'll be pulled by `makedepends` regardless.
So: no, no kittens will be harmed.